tensorflow/compiler/mlir/hlo/include/mlir-hlo/Dialect/mhlo/transforms/lmhlo_passes.td

```diff
@@ -51,3 +51,12 @@ def LhloLegalizeToParallelLoopsPass : Pass<"lhlo-legalize-to-parallel-loops", "F
   let constructor = "createLegalizeLhloToParallelLoopsPass()";
 }
 
+def LhloLegalizeRootsToParallelLoops : Pass<"lhlo-legalize-roots-to-parallel-loops", "FuncOp"> {
+  let summary = "Legalize the roots of lmhlo.fusion to parallel loops.";
+  let constructor = "createLhloLegalizeRootsToParallelLoopsPass()";
+}
+
+def InputInlineFusion : Pass<"input-inline-fusion", "FuncOp"> {
+  let summary = "Perform input inline fusion for fusion codegen";
```

Comment:
Can you provide a more complete documentation in the `let description` field?
Right now I didn't review in detail and I think it'd help with more doc on the intent and the invariant (before/after) for this pass.

Reply:
I add some documentation into the comments of input_inline_fusion_pass.cc. Pls have a look
(Seems no other passes use the description field for documentation? )
Pls let me know if you feel description field is better for documentation.